templates/repo/view_file.tmpl

```diff
@@ -111,6 +111,15 @@
 						{{end}}
 					</tbody>
 				</table>
+				<div class="code-view-menu-list ui fluid popup transition hidden">
+					<div class="ui column relaxed equal height">
+						<div class="column">
+							<div class="ui link list">
+								<a class="item ref-in-new-issue" href="{{.RepoLink}}/issues/new?permalink={{.AppURLTmp}}{{.RepoLink}}/src/commit/{{.CommitID}}/{{EscapePound .TreePath}}">{{.i18n.Tr "repo.issues.context.reference_issue"}}</a>
```

Comment:
There has to be a better way of getting the URL to the repo than doing this

{{.AppURLTmp}}{{.RepoLink}}


I'll have a look.

Also use the body query parameter.

Reply:
`{{AppUrl}}{{AppSubUrl}}{{.RepoLink}}` is what you're looking for, these are global template helpers.
Though I see the issue that AppUrl has a trailing `/` and .RepoLink has a leading `/`.. so more like
```
{{URLJoin AppUrl AppSubUrl .RepoLink}}
```

Reply:
Thx, fixed.